target.py

```diff
@@ -62,22 +63,15 @@ def load_single_type_targets(
         )
     ]
 
-
-def load_multiple_types_targets() -> list[Target]:
-    """
-    This function loads a list of target with different sprites.
-
-    :return: list of target (list[Target])
-    """
-    # @NOTE:
-    # This function can be expanded further once multiple
-    #  types of TARGETs are implemented.
-    return [
-        Target(
-            random.choice(
-                settings.TARGET["SPRITES"]
-            )
-        ) for _ in range(
-            settings.TARGET["TARGET_COUNT"]
-        )
-    ]
+    for target1 in targets:
```

Comment:
dit is nu ineens onderdeel van de functie heirboven?

Also, waarom target1 en 2? dat is lelijk